generate.sh

```diff
@@ -76,7 +76,7 @@ if [ -z "$ON_HEROKU" ]; then
 fi
 
 # Setting environment variables
-ENVVARS="$(python ${BASE}/modules/scripts/config.py "${OWNER}" "${REPONAME}")"
```

Comment:
What was the role of owner and reponame in the previous implementation? Just curious